lib/v2/datasource/remote/model/fiat_rate_model.dart

```diff
@@ -25,7 +24,7 @@ class FiatRateModel {
       model.rebase("USD");
       return model;
     } else {
-      return FiatRateModel(null, error: true);
+      return FiatRateModel(null);
```

Comment:
instead of returning an object that has null inside, why dont we just return null? 

Then if its null, we know something went wrong.